nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -311,7 +311,7 @@ let
         ${drivesCmdLine config.virtualisation.qemu.drives} \
         ${concatStringsSep " \\\n    " config.virtualisation.qemu.options} \
         $QEMU_OPTS \
-        "$@"
+        $@
```

Comment:
`"$@"` is special syntax in shell. It does not combine the arguments into a single argument.

Reply:
You are right, on further inspection that does not cause a problem that I thought I had, but it is strange to quote `$@` but not `$QEMU_OPTS`.